src/platform/Linux/BLEManagerImpl.h

```diff
@@ -170,9 +189,10 @@ class BLEManagerImpl final : public BLEManager, private BleLayer, private BlePla
 
     CHIPoBLEServiceMode mServiceMode;
     BLEAdvConfig mBLEAdvConfig;
+    BLEScanConfig mBLEScanConfig;
     uint16_t mFlags;
     char mDeviceName[kMaxDeviceNameLength + 1];
-    bool mIsCentral;
+    bool mIsCentral = false;
```

Comment:
Could you init mIsCentral in BLEManagerImpl::_Init()?

Reply:
I was planning mIsCentral is rather input of the _Init() function rather than its result, but I'll think about how to design tha better (how to let the application choose the BLE role)